scripts/sync_grafana_dashboards.sh

```diff
@@ -3,36 +3,45 @@
 # This script syncs the grafana dashboards from the grafana server to the local copy of this repo.
 # It is intended to be run from the root of the repo.
 
+DASHBOARD_FOLDER=${DASHBOARD_FOLDER:-aptos-core}
+GRAFANA_URL=${GRAFANA_URL:-https://o11y.aptosdev.com}
+LOCAL_DASHBOARD_FOLDER=${LOCAL_DASHBOARD_FOLDER:-"./dashboards"}
+
 ### Install grafana-sync tool
-if [[ "$(uname)" == "Darwin" ]]; then
-    wget https://github.com/mpostument/grafana-sync/releases/download/1.4.8/grafana-sync_1.4.8_Darwin_x86_64.tar.gz
-    sha=$(shasum -a 256 grafana-sync_1.4.8_Darwin_x86_64.tar.gz | awk '{ print $1 }')
-    [ "$sha" != "64be888acf049dea9485f002ee38e5a597f35a9b9ed7913cfbfd163747694c2c" ] && echo "shasum mismatch" && exit 1
-    tar -xvf grafana-sync_1.4.8_Darwin_x86_64.tar.gz grafana-sync
-else # Assume Linux
-    wget https://github.com/mpostument/grafana-sync/releases/download/1.4.8/grafana-sync_1.4.8_Linux_x86_64.tar.gz
-    sha=$(shasum -a 256 grafana-sync_1.4.8_Linux_x86_64.tar.gz | awk '{ print $1 }')
-    [ "$sha" != "c1b5a2c0d2b081d8acffaa06aebc83bca7cd47fdc8f3e7b4c252952b4fe15ec0" ] && echo "shasum mismatch" && exit 1
-    tar -xvf grafana-sync_1.4.8_Linux_x86_64.tar.gz grafana-sync
+if ! command -v grafana-sync &>/dev/null; then
+    echo "grafana-sync could not be found"
+    echo "installing..."
+    if [[ "$(uname)" == "Darwin" ]]; then
+        wget https://github.com/mpostument/grafana-sync/releases/download/1.4.8/grafana-sync_1.4.8_Darwin_x86_64.tar.gz
```

Comment:
This might be worth rewriting in python

Reply:
yea if it gets any more complicated, like needing to pull alerts, i'll do it in python (or typescript)...